### ISSUE TITLE
Allow to run ddccontrol without daemon

### ISSUE THREAD

**Issue by kravemir** [ddccontrol/ddccontrol#37]
For testing, and troubleshooting, purposes it would be useful to have an option to run ddccontrol without D-Bus daemon.

Behavior could be based environment variable `DDCCONTROL_NO_DAEMON`.

For example:

```
# as root user
DDCCONTROL_NO_DAEMON=1 ddccontrol [args...]

# as regular user
DDCCONTROL_NO_DAEMON=1 sudo ddccontrol [args...]
```

**Comment by kravemir** [comment]
Implemented in #38

**Comment by tg-x** [comment]
The help text documenting this is wrong: it suggests to set DDCCONTROL_NO_DBUS=1 instead of DDCCONTROL_NO_DAEMON
